bin/check-postgres-replication.rb

```diff
@@ -103,6 +103,9 @@ class CheckPostgresReplicationStatus < Sensu::Plugin::Check::CLI
   def run
     ssl_mode = config[:ssl] ? 'require' : 'prefer'
 
+    critical 'Slave host is undefined' unless config[:slave_host] && !['localhost', '127.0.0.1'].include?(config[:slave_host])
+    critical 'Master and slave cannot be the same host' unless config[:master_host] != config[:slave_host]
```

Comment:
This reads much cleaner to me, its better to assert positively than negatively and in this case you double asert negatively meaning its a positive assertion really:
```suggestion
    critical 'Master and slave cannot be the same host' if config[:master_host] == config[:slave_host]
```

Reply:
:+1: makes sense to me.